instance_control/bubuku_cluster.py

```diff
@@ -0,0 +1,74 @@
+import logging
+
+import click
+
+from instance_control.command.attach import AttachCommand
+from instance_control.command.create import CreateCommand
+from instance_control.command.get import GetCommand
+from instance_control.command.terminate import TerminateCommand
+from instance_control.command.upgrade import UpgradeCommand
+
+_LOG = logging.getLogger('bubuku.cluster.cli')
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.command('upgrade', help='Stop Kafka gracefully, detach esb volume, terminate the instance and launch a new one')
+@click.option('--cluster-name', required=True)
+@click.option('--image-version', help='Docker image version to use. By default the version from config is used. '
+                                      'If provided then verrides image version from config')
```

Comment:
for example you need to deploy different test version of image, why should you change files, you're just testing.